frontend/app/admin/src/views/settings/membership-types/components/PlatformMembershipTypePriceConfigEdit.vue

```diff
@@ -0,0 +1,136 @@
+<template>
+    <div class="container">
+        <hr>
+
+        <template v-if="hasMultiplePrices">
+            <h2 v-if="showStartDate" class="style-with-button">
+                <div>
+                    Prijs vanaf {{ $startDate ? formatDate($startDate) : '?' }}
+                </div>
+                <div>
+                    <button class="button text" type="button" @click="emits('delete')">
+                        <span class="icon trash" />
+                        <span class="hide-smartphone">{{ $t('Verwijderen') }}</span>
+                    </button>
+                </div>
+            </h2>
+            <h2 v-else>
+                Standaardprijs
+            </h2>
+        </template>
+
+        <STInputBox
+            v-if="showStartDate || $startDate"
+            :title="$t('admin.settings.membershipTypes.period.priceDate')" :error-box="errorBox"
+        >
+            <DateSelection
+                v-model="$startDate" :required="false"
+                :placeholder="$t('admin.settings.membershipTypes.expireDate.placeholder')"
+            />
+        </STInputBox>
+
+        <STList>
+            <STListItem v-for="[tagId, reduceablePrice] of model.prices" :key="tagId">
+                <PlatformMembershipTypeReduceablePriceEdit
+                    :model-value="reduceablePrice"
+                    :tag-id="tagId"
+                    :show-price-per-day="showPricePerDay"
+                    :error-box="errorBox" :validator="validator"
+                    @update:model-value="patchReduceablePrice(tagId, $event)"
+                >
+                    <STInputBox
+                        v-if="!tagId && (showPricePerDay || $pricePerDay)"
+                        title="Prijs per dag" :error-box="errorBox"
+                    >
+                        <PriceInput
```

Comment:
Dit klopt niet denk ik? Er is maar één prijs per dag, en deze staat hier per tag

Reply:
Ah, nvm